Back-End/server.js

```diff
@@ -0,0 +1,114 @@
+const dotenv = require("dotenv");
+dotenv.config();
+const express = require("express");
+const app = express();
+const uri = process.env.DATABASE_URI;
+const mongoOptions = { useUnifiedTopology: true };
+const mongodb = require("mongodb");
+const client = new mongodb.MongoClient(uri, mongoOptions);
+app.use(express.json());
+
+client.connect(() => {
+  const db = client.db("QUEUEANDA");
+  const collection = db.collection("Questions");
+
+  //get all questions in the Home Page
+  app.get("/api/questions", (request, response) => {
+    collection.find({}).toArray((error, questions) => {
+      if (error) {
+        response.status(500).send(error);
+      } else {
+        response.status(200).send(questions);
+      }
+    });
+  });
+
+  //get one specific question from home page  
+  app.get("/api/questions/question/:questionId",(request,response) => {
+    // params or body for id?  
```

Comment:
Think about the URL structures you used in https://syllabus.codeyourfuture.io/node/week-2/lesson - you usually use `<plural resource name>/<resource id>`, so this would be `/api/questions/:questionId`.